app/models/proxy_config.rb

```diff
@@ -103,8 +103,7 @@ def update_version
     config.update_all("version = 1 + (#{Arel.sql max_version.to_sql})")
 
     # Read the value
-    version = config.connection.select_value(config.select(:version)).to_i
-    write_attribute_without_type_cast 'version', version
+    self.version = config.connection.select_value(config.select(:version)).to_i
```

Comment:
```suggestion
    self.version = config.connection.select_value(config.select(:version)).to_i
    clear_attribute_changes [:version]
```

Reply:
@akostadinov it seemes to be working without clearing attribute changes. Do you think it's really needed?
`clear_attribute_changes` is also a private method... so I guess if we can avoid it - better?

Reply:
ah, yeah, you're right, I confused it with `clear_attribute_change(:attr)`

Reply:
except for the empty initializer file, I think this is my only outstanding request for the PR.